sdk/eventhub/azure-eventhubs/azure/eventhub/_consumer_producer_mixin.py

```diff
@@ -0,0 +1,133 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from __future__ import unicode_literals
+
+import logging
+import time
+
+from uamqp import errors, constants, compat
+from azure.eventhub.error import EventHubError, _handle_exception
+
+log = logging.getLogger(__name__)
+
+
+def _retry_decorator(to_be_wrapped_func):
+    def wrapped_func(self, *args, **kwargs):
+        timeout = kwargs.pop("timeout", None)
```

Comment:
Why not timeout = kwargs.pop("timeout", 100000)?

Reply:
good idea, will make the default value to 100000